biolearn/load.py

```diff
@@ -117,7 +117,7 @@ def load_nhanes(year):
         "LBXBAPCT",
     ]
     known_nhanes_year_suffix = {2010: "F", 2012: "G"}
-    if not known_nhanes_year_suffix[year]:
+    if not year in known_nhanes_year_suffix:
```

Comment:
[Optional] I think "x not in y" is easier to read.

Thank you for fixing this!

Reply:
Thanks for finding it! I agree on readability but since I plan to rewrite this part in the near future I'll make that change in a later PR assuming it's still applicable.